tools/add-badges.py

```diff
@@ -0,0 +1,71 @@
+import re
+import os
+import subprocess
+import fileinput
+import sys
+import shutil
+
+for repo_list_line in open('repositories.txt'):
+    m = re.search('(\d+)\|github.com/OCA/(.*)', repo_list_line)
+    repo_name = m.group(2)
+    repo_id = m.group(1)
+    print(repo_name)
+    if not os.path.exists(repo_name):
+        subprocess.call(['hub', 'clone', '--quiet', 'OCA/' + repo_name])
```

Comment:
What is the advantage of hub over git here? If it's only for not writing full URL, I prefer to put it and avoid the dependency.